grpc-ballerina/build.gradle

```diff
@@ -266,9 +267,13 @@ task ballerinaBuild {
                 commandLine 'sh', '-c', "$balJavaDebugParam $distributionBinPath/bal build ${testParams} ${debugParams}"
             }
         }
-        copy {
-            from file("$project.projectDir/target/bala")
-            into file("$artifactCacheParent/bala/${packageOrg}/${packageName}/${tomlVersion}")
+        // extract bala file to artifact cache directory
+        file("$project.projectDir/target/bala").eachFileMatch(~/.*.bala/) { balaFile ->
+            copy {
+                from zipTree(balaFile)
+                into file("$artifactCacheParent/bala/${packageOrg}/${packageName}/${tomlVersion}/${platform}")
+            }
+            delete(balaFile)
```

Comment:
@azinneera, since we are publishing the balas to the Ballerina central after this build, would this deletion cause a problem?

Reply:
Yes, Ill be removing this.

Reply:
Removed in 5bc376a. Also updated other rPRs of other repos